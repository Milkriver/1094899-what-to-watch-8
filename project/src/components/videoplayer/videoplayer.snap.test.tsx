import { render } from '@testing-library/react';
import VideoPlayer from './videoplayer';


describe('Component: VideoPlayer', () => {
  Object.defineProperty(HTMLMediaElement.prototype, 'muted', {
    set: jest.fn(),
    get: jest.fn(),
  });

  HTMLMediaElement.prototype.play = jest.fn();
  HTMLMediaElement.prototype.pause = jest.fn();

  test('should render correctly', () => {
    const page = (
      <VideoPlayer
        posterSrc='posterSrc'
        videoLink='videoLink'
      />
    );

    const { container } = render(page);
    expect(container).toMatchSnapshot();
  });
});
