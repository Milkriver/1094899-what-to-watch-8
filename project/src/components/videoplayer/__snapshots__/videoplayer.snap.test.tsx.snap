// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: VideoPlayer should render correctly 1`] = `
<div>
  <video
    height="200"
    poster="posterSrc"
    width="280"
  >
    <source
      src="videoLink"
      type="video/mp4"
    />
  </video>
</div>
`;
